### PR TITLE
change to singular for data types

### PR DESCRIPTION
**Pull request by yejee94**
changed

all options => option

### CODE REVIEW

**Review by timotheeguerin — COMMENTED** [review]
This doesn’t feel right to me there is multiple options

**Comment by yejee94** [comment]
@timotheeguerin this was suggested by Allen from last PR! I'm actually changing from options to option though??

**Review by allenjzhang — APPROVED** [review]

File: packages/typespec-azure-portal-core/lib/decorators.tsp [hunk 1]
```diff
@@ -71,8 +71,8 @@ model AboutOptions {
   learnMoreDocs?: string[];
 }
 
-/** Options for promotion of ARM resource. */
-model PromotionOptions {
+/** Option for promotion of ARM resource. */
+model PromotionOption {
```
Comment (timotheeguerin) [inline packages/typespec-azure-portal-core/lib/decorators.tsp:75]: I don't really get why this is a singlular options, there is 2? And the point of an option bag even if there is a single option is you can add more later which would be breaking if you had to change the name.

@allenjzhang can you expand on why you wanted this?

Comment (allenjzhang) [reply packages/typespec-azure-portal-core/lib/decorators.tsp:75]: My original thinking was on `xxxOptions` like `PromotionOptions` as it has a single one apiVersion settings. And plural of options is addressed as an [], so a `plural[]` sounded weird.

Also I had mentally thought of `xxOption` as a single option model for `xx` with bunch of properties (ie `@service`) rather a bunch of discret/independent option property bag grouped under `xx`. **But I do see your point on expandability and the names, particularly they have an `option` suffix. xxOptions sounds ok and reflect the multiple items under it. I am ok to take back my original comment.**

**Pull request closed without merging**